### ISSUE TITLE
Tinker With a Neural Network Right Here in Your Browser. 

### ISSUE THREAD

**Issue by Madonahs** [Madonahs/Machine-Learning#21]
**A nice Playground.** 

Click the Link. 

![pg](https://user-images.githubusercontent.com/11560987/39408848-578596c8-4ba2-11e8-9164-0bf4974913eb.PNG)

https://playground.tensorflow.org/#activation=tanh&batchSize=10&dataset=circle&regDataset=reg-plane&learningRate=0.03&regularizationRate=0&noise=0&networkShape=3,2&seed=0.88257&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=true&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false&hideText=false
